spec/controllers/idv/gpo_verify_controller_spec.rb

```diff
@@ -203,8 +203,9 @@
             create(
               :profile,
               :with_pii,
+              :fraud_review_pending,
+              fraud_pending_reason: 'threatmetrix_reject',
```

Comment:
Would it work to make the :fraud_review_pending attribute include setting fraud_pending_reason ?

Reply:
`fraud_review_pending` does set `fraud_pending_reason`. The reason for this is the description on the spec specifically says "when the threatmetrix status is reject" even though functionally the result is the same.